### PR TITLE
getStats es6 syntax

### PR DESCRIPTION
**Pull request by schayapathi**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### Description

Declaring default values in function declaration, object/array destructuring, concise require statements

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [ ] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### CODE REVIEW

File: lib/webrtc/getstats.js [hunk 1]
```diff
@@ -124,24 +119,24 @@ function standardizeChromeOrSafariActiveIceCandidatePairStats(stats) {
     { key: 'url', type: 'string' }
   ];
 
-  var standardizedLocalCandidateStatsKeys = standardizedCandidateStatsKeys.concat([
+  const standardizedLocalCandidateStatsKeys = standardizedCandidateStatsKeys.concat([
```
Comment (PikaJoyce) [inline lib/webrtc/getstats.js:122]: We can use the spread operator here instead of concat.

File: lib/webrtc/getstats.js [hunk 2]
```diff
@@ -206,41 +201,37 @@ function standardizeFirefoxActiveIceCandidatePairStats(stats) {
...
```
Comment (PikaJoyce) [inline lib/webrtc/getstats.js:204]: We can use the spread operator here instead of concat.

**Review by PikaJoyce — CHANGES_REQUESTED** [review]

File: lib/webrtc/getstats.js [hunk 3]
```diff
@@ -35,43 +32,43 @@ function getStats(peerConnection, options) {
...
     [remoteVideoTracks, 'remoteVideoTrackStats', true]
-  ], function(triple) {
-    var tracks = triple[0];
-    var statsArrayName = triple[1];
-    var isRemote = triple[2];
-    return tracks.map(function(track) {
+  ], triple => {
+    const tracks = triple[0];
+    const statsArrayName = triple[1];
+    const isRemote = triple[2];
```
Comment (manjeshbhargav) [inline lib/webrtc/getstats.js:56]:
```suggestion
  ], ([tracks, statsArrayName, isRemote]) => {
```

File: lib/webrtc/getstats.js [hunk 4]
```diff
@@ -486,54 +472,54 @@ function standardizeChromeOrSafariStats(response) {
...
       source, // local rtp stats
       localMedia,
       track,
       codec,
       remoteSource && remoteSource.ssrc === source.ssrc ? remoteSource : null, // remote rtp stats
     ];
 
-    function getStatValue(name) {
-      var sourceFound = statSources.find(function(statSource) {
+    const getStatValue = name => {
```
Comment (manjeshbhargav) [inline lib/webrtc/getstats.js:490]: Revert to function.

**Review by manjeshbhargav — APPROVED** [review]

**Pull request merged**